docs/troubleshooting/README.md

```diff
@@ -1 +1,611 @@
+        reason: AppliedManifestFailedReason
+        status: "False"
+        type: Applied
+      kind: Namespace
+      name: test-ns
+      version: v1
+  - clusterName: kind-cluster-2
+    conditions:
+    - lastTransitionTime: "2023-11-28T20:56:15Z"
+      message: 'Successfully scheduled resources for placement in kind-cluster-2:
+        picked by scheduling policy'
+      observedGeneration: 2
+      reason: ScheduleSucceeded
+      status: "True"
+      type: ResourceScheduled
+    - lastTransitionTime: "2023-11-28T20:56:15Z"
+      message: Successfully Synchronized work(s) for placement
+      observedGeneration: 2
+      reason: WorkSynchronizeSucceeded
+      status: "True"
+      type: WorkSynchronized
+    - lastTransitionTime: "2023-11-28T20:56:21Z"
+      message: Successfully applied resources
+      observedGeneration: 2
+      reason: ApplySucceeded
+      status: "True"
+      type: ResourceApplied
+  selectedResources:
+  - group: apps
+    kind: Deployment
+    name: test-nginx
+    namespace: test-ns
+    version: v1
+  - kind: Namespace
+    name: test-ns
+    version: v1
+```
+
+in the **placementStatuses** section of the **CRP status** for kind-cluster-1 in **failedPlacements** we get a clear message as to why the resource failed to apply on the member cluster.
+
+At times, we might need more information in that case please take a look at the work object
+
+**work status:**
+```
+ status:
+    conditions:
+    - lastTransitionTime: "2023-11-28T21:07:15Z"
+      message: Failed to apply work
+      observedGeneration: 1
+      reason: AppliedWorkFailed
+      status: "False"
+      type: Applied
+    manifestConditions:
+    - conditions:
+      - lastTransitionTime: "2023-11-28T20:56:16Z"
+        message: ManifestNoChange
+        observedGeneration: 1
+        reason: ManifestNoChange
+        status: "True"
+        type: Applied
+      identifier:
+        group: apps
+        kind: Deployment
+        name: test-nginx
+        namespace: test-ns
+        ordinal: 0
+        resource: deployments
+        version: v1
+    - conditions:
+      - lastTransitionTime: "2023-11-28T20:56:16Z"
+        message: 'Failed to apply manifest: resource is not managed by the work controller'
+        reason: AppliedManifestFailedReason
+        status: "False"
+        type: Applied
+      identifier:
+        kind: Namespace
+        name: test-ns
+        ordinal: 1
+        resource: namespaces
+        version: v1
+```
+
+from looking at the **work status** and specifically the **manifestConditions** section we could see that the namespace could not be applied but the deployment within the namespace got propagated from hub to the member cluster correctly. In this case to solve this issue maybe delete the existing namespace on the member cluster but that's upto the user to decide since the namespace could already contain resources within it.
```

Comment:
```suggestion
From looking at the **work status** and specifically the **manifestConditions** section we could see that the namespace could not be applied but the deployment within the namespace got propagated from hub to the member cluster correctly. In this case to solve this issue maybe delete the existing namespace on the member cluster but that's upto the user to decide since the namespace could already contain resources within it.
```